src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -5,13 +5,31 @@
     <Description>Monitor.Tests Class library</Description>
     <PackageId>Monitor.Tests</PackageId>
     <VersionPrefix>1.0.0-preview</VersionPrefix>
-    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452</TargetFrameworks>
+    <!-- TargetFrameworks>net452;netcoreapp1.1</TargetFrameworks -->
+  </PropertyGroup>
+
+  <ItemGroup>
+	  <!-- PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.0.0" / >
+	  <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.8" / >
+	  <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.7.2" / -->
```

Comment:
@gucalder if you see the difference between those two nugets, you will see the namespace is different.
So all I was trying to say is to use the correct namespace and types from the 1.6.0 version